### PR TITLE
Bug 658052: Search page does not validate

### PR DESCRIPTION
**Pull request by rfreebern**

### CODE REVIEW

File: apps/search/views.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def results(request):
         data = _query(q, user=request.user)
         return render(request, 'results.html', data)
     else:
-        return render(request, 'blank.html')
+        return render(request, 'blank.html', {'page': 'search'})
```
Comment (seanmonstar) [inline apps/search/views.py:82]: whats this for?

Comment (rfreebern) [inline apps/search/views.py:82]: It sets the &lt;body&gt; id to 'search'. Without it, the HTML output was &lt;body id=''&gt; which the validator was marking invalid.

Comment (seanmonstar) [inline apps/search/views.py:82]: Ah ok. Then this line is fine, but perhaps in the template where `<body id="{{ page }}">` it should be wrapped in a conditional. That way, any other page that has forgotten to add a "page" value to its data won't also be wrong.

`<body {% if page %}id="{{page}}"{% endif %}>`

Comment (rfreebern) [inline apps/search/views.py:82]: Thanks, Sean. I've added that to the branch.

**Pull request merged**